content/download.html

```diff
@@ -147,6 +155,7 @@ <h3>Community-maintained packages</h3>
       <pre>$ xbps-install openra</pre>
     </dd>
   </dl>
+  <p class="downloadthirdparty">We do not maintain this external package source, so there may be delays when a new version is released.<br />Please contact the downstream repository maintainers about any packaging issues.</p>
```

Comment:
Please change "this" &rarr; "these" and "source" &rarr; "sources". Should be good to merge then.

Reply:
Done.